### ISSUE TITLE
Implementing changes for playerclass demo

### ISSUE THREAD

**Issue by katie-gao** [uchicago-cs/chiventure#1545]
As referenced in the previous issue, Van, Jake, and I plan to see what changes are needed to do a two playerclass multiclass demo.  This issue will discuss the current issues we have, and figuring out what the previous year did to successfully demo the warrior class.

**Comment by vanolson** [comment]
Katie and I are evaluating how much of last year's demo can be repurposed, which is located [here](https://github.com/uchicago-cs/chiventure/tree/dev/src/playerclass/examples).

**Comment by katie-gao** [comment]
A lot of last year's demo can be used, but this year we want to focus on the new classes skills plus the revamped multiclass module that our team has worked on all quarter. As we are running last year's demo and understanding the code, we are also going to see where we can better. At first glance, we noticed a few typos with their demo and functions that need to be restructured to match the changes we have made this past quarter.

**Comment by katie-gao** [comment]
Jake is making some changes to the multi class section of the print_class function

**Comment by wellingtj** [comment]
Currently making a multiclass skill PR, this will be a key feature of the demo ideally.

**Comment by katie-gao** [comment]
We have figured out the executable, the make functions, and have a kinda-ish demo for the prefabs.

**Comment by vanolson** [comment]
We've written a brief display skill trees function to use to show our newly created prefab skill trees.

**Comment by vanolson** [comment]
The prefab display portion of the demo code looks to be in pretty good shape, with the new updates made to the display skill trees function. What's remaining is some of the multiclass portion, though that's in development as well.

**Comment by wellingtj** [comment]
We have made significant progress on the multiclass portion, where there are now two new files that include multiclass_prefabs. One is 'include/playerclass/multiclass_prefabs.h' and the other is at 'src/playerclass/src/multiclass_prefabs.c'. We created three new multiclasses alongside the hexblade (warrior and wizard) multiclass that last year had. There is much more functionality with multiclass now as 'multiclass.h' is now fully functional and updated. This was the big display and change of the demo because now we can actually use the multiclass module properly. The three additional multiclasses are "infernal" (elementalist, sorcerer), "paladin" (knight, monk), "enchanter" (bard, healer), and "assassin" (rogue, ranger).
